claude-product-cycle/commands-layer/client.md

```
@@ -0,0 +1,156 @@
+# /client - Client Layer Implementation
+
+## Purpose
+Implement the client layer (Network + Data) for a feature. This includes DTOs, Services, and Repositories.
+
+---
+
+## Workflow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)